azurelinuxagent/ga/env.py

```diff
@@ -80,6 +81,13 @@ def run(self):
         self.dhcp_handler.conf_routes()
         self.hostname = self.osutil.get_hostname_record()
         self.dhcp_id_list = self.get_dhcp_client_pid()
+        # Cleanup MDS firewall rule and ensure WireServer firewall rule is set
```

Comment:
the env thread is meant to check for required  configuration (e.g. wireserver firewall rules) periodically.
isn't the migration supposed to be happen only 1 time?